Sources/DiagnosticsLogger.swift

```diff
@@ -173,7 +183,7 @@ extension DiagnosticsLogger {
     private func log(message: String, file: String = #file, function: String = #function, line: UInt = #line) {
         guard isSetup else { return assertionFailure("Trying to log a message while not set up") }
 
-        self.queue.async { [unowned self] in
+        queue.async { [unowned self] in
```

Comment:
<!--
  0 failure: 
  1 warning:  It is safer to us...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DiagnosticsLogger.swift;
  Line: 186;
-->

- :warning: It is safer to use weak instead of unowned